scripts/mantidpython.py

```diff
@@ -3,45 +3,156 @@
 import re
 import subprocess
 import sys
+import tempfile
 
-mantidRegExp=re.compile(r'/opt/.antid.*/bin')
-def getMantidLoc(line):
+mantidRegExp = re.compile(r'/opt/.antid.*/bin')
+
+MANDID_VERSION = 'MANTID_VERSION='
```

Comment:
The strings you are looking for shouldn't include the `=`. People will likely put `CONDA_ENV =` and `CONDA_ENV=` and expect the same thing.

Reply:
Applied with more tests.